modules/dxp/apps/segments/segments-asah-connector/src/main/java/com/liferay/segments/asah/connector/internal/expression/FilterByCountIndividualSegmentsExpressionVisitorImpl.java

```diff
@@ -285,6 +346,34 @@ else if (object == null) {
 		return object;
 	}
 
+	private FilterByCount.Day _getFilterByCountDay(
```

Comment:
As this code is called only once, I think it is safe to move it to the visitFunctionCallExpression.